docs/resources/artifactory_local_npm_repository.md

```diff
@@ -0,0 +1,17 @@
+# Artifactory Local npm Repository Resource
+
+Creates a local npm repository and allows for the creation of a 
+
+## Example Usage
+
+```hcl
+resource "artifactory_local_npm_repository" "terraform-local-test-npm-repo-basic" {
```

Comment:
It's important to note in the documentation that there isn't anything different from a generic repo. Please add information to that effect here

Reply:
sure.